packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -26,7 +26,7 @@ import { useTheme } from '@/hooks/use-theme';
 import * as detailsStyles from '@/scss/details.module.scss';
 
 import * as styles from './index.module.scss';
-import type { ApiResourceDetailsOutletContext } from './types';
+import { ApiResourceDetailsOutletContext } from './types';
```

Comment:
I'm curious why the `type` is removed...

Reply:
It is removed by the eslint...
<img width="850" alt="image" src="https://user-images.githubusercontent.com/10806653/217252784-deebf684-4159-446c-8c28-5725f6cf014d.png">